src/SDKs/Azure.Base/data-plane/Azure.Base/Http/Pipeline/FixedPolicy.cs

```diff
@@ -5,12 +5,12 @@
 
 namespace Azure.Base.Http.Pipeline
 {
-    class FixedPolicy : RetryPolicy {
+    public class FixedRetryPolicy : RetryPolicy {
```

Comment:
nit:  I know this is marked as [No Merge] but the class and filenames have diverged.   You may want to consider renaming the file, just in case...

Reply:
Yeah, I will clean it up. In fact, after I am happy with the result, I will just close this PR and do all its elements in several separate PRs. This PR just bundles several independent features.